akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -207,6 +207,8 @@ not exist in the class.
 
 ### Add Field
 
+#### Add Optional Field
```

Comment:
maybe too many levels. would be enough with `### Add Optional Field` and `### Add Mandatory Field`

Reply:
There's some link to `#add-field` from another page and I just preferred keeping the title.